lib-android/src/main/res/values/attrs_layoutviewpager.xml

```diff
@@ -1,9 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should consider one naming convention (attrs_iconTabLayout.xml,  attrs_layoutviewpager.xml). Camel case looks better IMO.